source/Calamari.Common/Commands/RunningDeployment.cs

```diff
@@ -7,9 +7,6 @@ namespace Calamari.Common.Commands
 {
     public class RunningDeployment
     {
-        public delegate RunningDeployment Factory(string? packageFilePath,
-            Dictionary<string, string>? environmentVariables = null);
```

Comment:
Because the tests are now executing calamari out-of-process, we can specify the environment variables for the executable before it runs so we don't need to do any fance in-process passing in specific environment variables anymore. This means that this previous change to RunningDeployment can be removed and we can new it up like we were originally.

Reply:
This also means we no longer need to register `RunningDeployment` in the Calamari Flavour Program classes.